### PR TITLE
Entrypoint

### PR DESCRIPTION
**Pull request by ardiesaeidi**
Added `EntryPoint` to spec to provide a resource for clients to be able to learn more about the API as well as provide an entry point.

### CODE REVIEW

File: _specs/1.0/index.md [hunk 1]
```diff
@@ -277,10 +277,58 @@ A `Collection` **MAY** have the following:
...
+* `@links`: Repesents a [link](#document-components-link-collection) with navigational _links_ to other APIs and additional resources.
+    * `documentation`: Represents a [link value](#document-components-link-value) navigating to the documentation page.
+    * `support`: Represents a [link value](#document-components-link-value) navigating to the support page.
+
+* `name`: Represents the name of the API as string.
+
+A `EntryPoint` **MAY** have the following:
+
+* `description`: Represents a description of the API as string.
+* `version`: Represents the current version of the API following [SEMVER](https://semver.org) as string.
```
Comment (thehelix112) [inline _specs/1.0/index.md]: I don't think we're using semver.org strictly.  It specifics it all must be X.Y.Z, where we are more open to something like 'v1'  Maybe we can point people at the url design section instead?

**Review by thehelix112 — CHANGES_REQUESTED** [review]
Update semver reference and fix grammar typo and then good.

File: _specs/1.0/index.md (hunk 1 above)
Comment (ardiesaeidi) [reply _specs/1.0/index.md]: Ah yes i agree, i'll make that change

File: _specs/1.0/index.md [hunk 2]
```diff
@@ -277,10 +277,58 @@ A `Collection` **MAY** have the following:
...
 
-Processing errors can be handled by returning a [node object](#document-components-node-object) of `Error` to consumers. In addition, an appropriate HTTP status code, a human readable `code` must be returned.
+It **MUST** be the top most JSON object and **MUST NOT** be nested.
+
+A `EntryPoint` **MUST** have the following:
+
+* `@id`: Represents a valid [URI](#conventions-uri).
+* `@type`: Have a value of `EntryPoint`.
+* `@links`: Repesents a [link](#document-components-link-collection) with navigational _links_ to other APIs and additional resources.
+    * `documentation`: Represents a [link value](#document-components-link-value) navigating to the documentation page.
```
Comment (ardiesaeidi) [inline _specs/1.0/index.md]: I'll make this optional for the time being as we devs might not have a docs/support site ready.

**Pull request merged**